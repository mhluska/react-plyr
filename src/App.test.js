import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Plyr from './index';

it('renders a simple Youtube Video', () => {
  const wrapper = shallow(
    <Plyr
      type="youtube"
      videoId="CDFN1VatiJA"
    />
  );
  expect(toJson(wrapper)).toMatchSnapshot();
});

it('renders a simple Vimeo Video', () => {
  const wrapper = shallow(
    <Plyr
      type="vimeo"
      videoId="https://vimeo.com/189789787"
    />
  );
  expect(toJson(wrapper)).toMatchSnapshot();
});
