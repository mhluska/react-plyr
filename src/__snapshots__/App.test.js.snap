// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a simple Vimeo Video 1`] = `
<div
  className="react-plyr"
>
  <div
    data-type="vimeo"
    data-video-id="https://vimeo.com/189789787"
  />
</div>
`;

exports[`renders a simple Youtube Video 1`] = `
<div
  className="react-plyr"
>
  <div
    data-type="youtube"
    data-video-id="CDFN1VatiJA"
  />
</div>
`;
